packages/components/src/box-control/axial-input-controls.tsx

```diff
@@ -129,25 +145,45 @@ export default function AxialInputControls( {
 						? selectedUnits.top
 						: selectedUnits.left;
 				return (
-					<UnitControl
-						{ ...props }
-						isFirst={ first === side }
-						isLast={ last === side }
-						isOnly={ only === side }
-						value={ [
-							parsedQuantity,
-							selectedUnit ?? parsedUnit,
-						].join( '' ) }
-						onChange={ createHandleOnChange( side ) }
-						onUnitChange={ createHandleOnUnitChange( side ) }
-						onFocus={ createHandleOnFocus( side ) }
-						onHoverOn={ createHandleOnHoverOn( side ) }
-						onHoverOff={ createHandleOnHoverOff( side ) }
-						label={ LABELS[ side ] }
-						key={ side }
-					/>
+					<HStack key={ `box-control-${ side }` }>
+						<FlexedBoxControlIcon side={ side } sides={ sides } />
+						<UnitControl
+							{ ...props }
+							isFirst={ first === side }
+							isLast={ last === side }
+							isOnly={ only === side }
+							value={ [
+								parsedQuantity,
+								selectedUnit ?? parsedUnit,
+							].join( '' ) }
+							onChange={ createHandleOnChange( side ) }
+							onUnitChange={ createHandleOnUnitChange( side ) }
+							onFocus={ createHandleOnFocus( side ) }
+							onHoverOn={ createHandleOnHoverOn( side ) }
+							onHoverOff={ createHandleOnHoverOff( side ) }
+							label={ LABELS[ side ] }
+							key={ side }
+						/>
+						<FlexedRangeControl
+							__nextHasNoMarginBottom
+							hideLabelFromVision
+							initialPosition={ 0 }
```

Comment:
If we don't, then the `RangeControl` thumb starts in the center of the min and max:

<img width="666" alt="Screenshot 2023-12-08 at 1 27 09 PM" src="https://github.com/WordPress/gutenberg/assets/35543432/4292ecb7-eb2a-4e51-812d-0c6297c484fc">

This seems a bit confusing since the UnitControl input is empty initially.

Reply:
We discussed on a call that having `initialPosition={ parsedQuantity ?? 0 }` would be a good solution here.

Reply:
Apologies if I didn't explain myself clearly. My suggestion was to remove the `initialPosition` prop, and instead have `value={ parsedQuantity ?? 0 }`.

That way we are always using the `value` prop, which is the prop that should be used to use the component in controlled mode (as opposed to `initialPosition`, which is supposed to be used to set the initial value when the component is used in uncontrolled mode, as per the [docs](https://github.com/WordPress/gutenberg/blob/da4d91f8e0077910252c7c0feb3c5e67ec335d6d/packages/components/src/range-control/types.ts#L121-L126))

Reply:
No worries, thank you for clarifying! I'll make the change 👍